Services/DataBaseService/DbModels/DbUserCredential.cs

```diff
@@ -9,6 +9,7 @@ public class DbUserCredential
         public Guid Id { get; set; }
         public string Email { get; set; }
         public string PasswordHash { get; set; }
+        public bool IsExist { get; set; }
```

Comment:
Exists mb?

Reply:
Or IsActive (as Kirill said below)

Reply:
I have thought about it, but it can conflict with online user or active (when it execute other operation)

Reply:
> I have thought about it, but it can conflict with online user or active (when it execute other operation)

Then come up with any other option which follows english grammar

Reply:
For example 'Exists' as I said above